src/ops/ops.go

```diff
@@ -368,11 +369,13 @@ func (o *ops) renderDeploymentFiles(srcTemplate string, params map[string]interf
 	return nil
 }
 
-func (o *ops) GetVGByPV(pvName string) (string, error) {
+func (o *ops) GetVolumeGroupsByDisk(diskName string) ([]string, error) {
+	var VGs []string
```

Comment:
lowcase?

Reply:
I'm not sure it should be? every other method in the interface (many of these only used internally also) are Uppercase.

See 

https://github.com/openshift/assisted-installer/blob/master/src/ops/ops.go#L34-L62

Reply:
i mean "var VGs []string" should be "var vgs []string"

Reply:
agreed, and fixed